experimental/Schemes/src/elliptic_surface.jl

```diff
@@ -2296,11 +2314,13 @@ function _elliptic_surface_with_trafo(g::MPolyRingElem{<:AbstractAlgebra.Generic
 
   # The following three commands won't work unless we convert to a rational_function_field
   EE = base_change(x->evaluate(x, t), E)
-
-  EE = tates_algorithm_global(EE)
-  EE, _ = short_weierstrass_model(EE)
-  EE, _ = integral_model(EE)
-
+  @show EE
```

Comment:
```suggestion
```
just a reminder to remove this `show` before merging

Reply:
Thank you! Got it.